DQMOffline/Trigger/python/ParticleNetAK4BTagClient_cfi.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+particleNetAK4BTagEfficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs    = cms.untracked.vstring("HLT/HIG/PNETAK4/HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_PFDiJet30_PFBTagParticleNet_2BTagSum0p65/"),
+    verbose    = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution = cms.vstring(),
+    efficiency = cms.vstring(
+        "eff_muon_pt 'Efficiency vs p_{T}(#mu); p_{T}(#mu); efficiency' muon_pt_numerator muon_pt_denominator",
+        "eff_muon_eta 'Efficiency vs #eta(#mu); #eta(#mu); efficiency' muon_eta_numerator muon_eta_denominator",
+        "eff_electron_pt 'Efficiency vs p_{T}(ele); p_{T}(ele); efficiency' electron_pt_numerator electron_pt_denominator",
+        "eff_electron_eta 'Efficiency vs #eta(ele); #eta(ele); efficiency' electron_eta_numerator electron_eta_denominator",
+        "eff_dilepton_pt 'Efficiency vs p_{T}(ll); p_{T}(ll); efficiency' dilepton_pt_numerator dilepton_pt_denominator",
+        "eff_dilepton_mass 'Efficiency vs m(ll); m(ll); efficiency' dilepton_mass_numerator dilepton_mass_denominator",
+        "eff_ht 'Efficiency vs H_{T}; H_{T}; efficiency' ht_numerator ht_denominator",
```

Comment:
This line in `master` is currently missing a comma, and I guess this fix might lead to a few differences in PR outputs.
https://github.com/cms-sw/cmssw/blob/96a47c5f5d7bf85549ddcbaba0ec6fb9360c80ca/DQMOffline/Trigger/python/ParticleNetJetTag_Client_cfi.py#L15